modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -86,7 +86,6 @@ spec:
       enabled: false # <2>
     kiali:
       name: kiali-user-workload-monitoring
-spec:
   meshConfig:
     extenstionProviders:
```

Comment:
Typo:
```suggestion
    extensionProviders:
```

Reply:
Eagle Eyes @luksa! Thanks Marko, I only looked at the line called out in the story and didn't even see this in the next line. Appreciate your diligence!

Change has been submitted.